### ISSUE TITLE
Support images for participants

### ISSUE THREAD

**Issue by Drarig29** [Drarig29/brackets-viewer.js#36]
Example of use (taken from [here](https://dota2classic.ru/tournament/41/bracket)):

![image](https://user-images.githubusercontent.com/9317502/110312717-c304de00-8005-11eb-9608-74072a5c043c.png)

I think a function should provide images (or `null`) for each participant before calling `render()`.

**Comment by qlaffont** [comment]
Actually I can handle click on match with this : 

```
document
        .querySelectorAll('[data-match-id]')
        .forEach((element) =>
          element.addEventListener('click', () =>
            console.log(element.getAttribute('data-match-id')),
          ),
        );
```

**Comment by Drarig29** [comment]
> Yes, and maybe to add an event listener onClick to handle :
> 
> * Click on participant
> * Click on match

This is related to #38.

And no, I won't add an event listener for a click on a participant because it will be mistaken for a click on a match. So I'll only add a listener for a click on a match. And you, in a match page for example, can do something to handle clicks on a participant afterwards.

It works like this in Toornament, and I like it.

**Comment by Drarig29** [comment]
Closes #37

**Comment by Drarig29** [comment]
> Actually I can handle click on match with this :
> 
> ```
> document
>         .querySelectorAll('[data-match-id]')
>         .forEach((element) =>
>           element.addEventListener('click', () =>
>             console.log(element.getAttribute('data-match-id')),
>           ),
>         );
> ```

You can now use https://github.com/Drarig29/brackets-viewer.js/commit/ae66b72b505cc273e78ca3d089d03f75c027f48b
